qa/integration-tests-webapps/src/test/java/org/camunda/bpm/CsrfPreventionIT.java

```diff
@@ -26,25 +26,20 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 
-public class CsrfPreventionIT extends AbstractWebappIntegrationTest {
-
-  @Override
-  protected String getApplicationContextPath() {
-    return "camunda/app/tasklist/default/";
-  }
+public class CsrfPreventionIT extends AbstractWebIntegrationTest {
 
   @Before
-  public void preventRaceConditions() throws InterruptedException {
-    // just wait some seconds before starting because of Wildfly / Cargo race conditions
-    Thread.sleep(5 * 1000);
+  public void createClient() throws Exception {
+    preventRaceConditions();
+    createClient(getWebappCtxPath());
```

Comment:
I think this should be:
```suggestion
    super.createClient(getWebappCtxPath());
```